src/pseudopeople/configuration/validator.py

```diff
@@ -19,6 +19,12 @@ def validate_overrides(overrides: Dict, default_config: ConfigTree) -> None:
     if not isinstance(overrides, Dict):
         raise ConfigurationError("Invalid configuration type provided.") from None
     for dataset, dataset_config in overrides.items():
+        if not isinstance(dataset_config, Dict):
+            raise ConfigurationError(
+                f"'{dataset}' must be a Dict. "
+                f"Provided {dataset_config} of type {type(dataset_config)}."
+            )
+
```

Comment:
We would need test coverage for this.

Reply:
We have no test coverage of any of the validator, as far as I can tell. So I figured it would be out of scope to introduce that. If you'd prefer, I could add a test _only_ for the functionality in this PR and not for anything else in the validator?

Reply:
We indirectly do in `test_configuration.py`

Reply:
Good point! Not sure how I missed that. In that case, it's much more straightforward to add tests for this PR.